### PR TITLE
Make function definition consistent

### PR DESCRIPTION
**Pull request by nicolasbock**
We tend to define functions as `function foo()`, but not always. This patch
makes all function defitions follow this scheme consistently.

### CODE REVIEW

**Comment by bmwiedemann** [comment]
according to
http://tldp.org/HOWTO/Bash-Prog-Intro-HOWTO-8.html
http://ryanstutorials.net/bash-scripting-tutorial/bash-functions.php
it should either have () or the "function" keyword, but not both, so I guess we will need to do some more consistency changes here... but better do that after Dirk is done moving around code in #1263

**Comment by nicolasbock** [comment]
@bmwiedemann Good point. Let's wait. Which form do you prefer?

**Comment by dirkmueller** [comment]
go ahead..

**Comment by nicolasbock** [comment]
@dirkmueller Which of the two versions that @bmwiedemann found do you prefer?

1. `foo()`
2. `function foo`

**Comment by bmwiedemann** [comment]
`function foo` seems nicer - more explicit

**Comment by aspiers** [comment]
I'm not religious about the style either way, but I think this will be somewhat wasted effort unless:

1. you change the style across the whole repo (or at least the most commonly-used files)
2. (more importantly) you add CI which enforces the style. This could be solved by resurrecting #936.

**Comment by nicolasbock** [comment]
@aspiers I am a big fan of ShellCheck. My thumbs up for resurrecting it!

**Comment by nicolasbock** [comment]
@bmwiedemann and @aspiers  I updated the commit to switch to `function foo`. What do you think?

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3711,7 +3712,7 @@ function oncontroller_manila_generic_driver_setup()
         "echo \"ERROR: manila service VM not listening on ssh port. manila tests will fail!\""
 }
 
-function oncontroller_magnum_service_setup ()
+function oncontroller_magnum_service_setup
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: bashate is complaining here about Trailing Whitespace

Comment (nicolasbock) [inline scripts/qa_crowbarsetup.sh]: Fixed

**Comment by nicolasbock** [comment]
@jdsn :)

**Pull request merged**